client/src/components/StreamField/scss/components/c-sf-button.scss

```
@@ -1,64 +1,37 @@
 .c-sf-button {
-    $root: &;
-    display: flex;
     flex: 1 1 200px;
     margin: $add-panel-gutter;
     appearance: none;
+    color: #333;
+    line-height: 1.5;
```

The old buttons had a height of 42px, this results in 38px. 

```suggestion
    line-height: 1.833;
```

should bring it up to previous size